src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs

```diff
@@ -174,7 +174,13 @@ private static string GetTenantIdFromToken(SecurityToken securityToken)
         {
             if (securityToken is JwtSecurityToken jwtSecurityToken)
             {
-                if (jwtSecurityToken.Payload.TryGetValue(ClaimConstants.Tid, out object? tenantId))
+                if (jwtSecurityToken.Payload.TryGetValue(ClaimConstants.Tid, out object? tid))
```

Comment:
nit: not consistent with other if-conditions that are checking for tid/tenantId.

Reply:
@GeoK the `TryGetPayloadValue` isn't exposed on the `JwtSecurityToken`, like it is on the `JsonWebToken` object. hence the difference.